README.md

```diff
@@ -61,680 +56,31 @@ The easiest way to install LocalStack is via `pip`:
 pip install localstack
 ```
 
-**Note**: Please do **not** use `sudo` or the `root` user - LocalStack
-should be installed and started entirely under a local non-root user. If you have problems
-with permissions in MacOS X Sierra, install with `pip install --user localstack`
+**Note**: Please do **not** use `sudo` or the `root` user - LocalStack should be installed and started entirely under a local non-root user. If you have problems with permissions in MacOS X Sierra, install with `pip install --user localstack`
 
-## Running
-
-By default, LocalStack is started inside a Docker container by running:
```

Comment:
i think it makes sense to leave one example to start localstack via the cli, and maybe tease some of the experience that the person will have

something like
```
 % localstack start -d

     __                     _______ __             __
    / /   ____  _________ _/ / ___// /_____ ______/ /__
   / /   / __ \/ ___/ __ `/ /\__ \/ __/ __ `/ ___/ //_/
  / /___/ /_/ / /__/ /_/ / /___/ / /_/ /_/ / /__/ ,<
 /_____/\____/\___/\__,_/_//____/\__/\__,_/\___/_/|_|

 💻 LocalStack CLI 0.13.1

[20:22:20] starting LocalStack in Docker mode 🐳
[20:22:21] detaching
```
which will download and start the localstack docker container

you can query the state with:
```
 % localstack status         
┏━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ Runtime version ┃ 0.13.0.10                                             ┃
│ Docker image    │ tag: latest, id: 41d3a94e8776, 📆 2021-12-05T15:37:01 │
│ Runtime status  │ ✔ running (name: "localstack_main", IP: 172.17.0.2)   │
└─────────────────┴───────────────────────────────────────────────────────┘
```
or
```
 % localstack status services
┏━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┓
┃ Service                  ┃ Status      ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━┩
│ acm                      │ ✔ available │
│ apigateway               │ ✔ available │
│ cloudformation           │ ✔ available │
│ cloudwatch               │ ✔ available │
│ config                   │ ✔ available │
│ dynamodb                 │ ✔ available │
...
```

Reply:
I have created an `Example` section to exactly serve this purpose.